### ISSUE TITLE
use libm.a instead of libm.so on Linux Aarch64

### ISSUE THREAD

**Issue by jperedadnr** [gluonhq/substrate#919]
Linking the dynamic library causes 
```
./HelloFX: /lib/aarch64-linux-gnu/libm.so.6: version `GLIBC_2.29' not found (required by ./HelloFX) 
```
when running on embedded devices with older glibc versions, like Raspbian 64 bits.

Using the static version solves the issue.

**Comment by jperedadnr** [comment]
Fixed with #923

**Comment by alaa-az** [comment]
I am getting this problem with Linux-x86_64 with maven plugin 0.1.42
